notebooks

```diff
@@ -0,0 +1 @@
+docs/source/notebooks/
```

Comment:
Just so windows users don't have headaches, I'd suggest not using a symbolic link. Maybe just have a README file here that tells users where to find the notebooks? Or remove the notebooks directory entirely?

Reply:
I was mostly worried about hyperlinks to the notebook directory on github which might become stale without the symlink. But if you think that's not a big problem I can also remove it and add a line to the REAME or so.

Reply:
That's certainly a risk too. We can also just remove the notebooks directory entirely, everything is in the docs anyway...